index.js

```diff
@@ -18,7 +12,50 @@ function displayBooks(book, index) {
   return content;
 }
 
-function storeForm(form) {s
+function storeForm(form) {
   localStorage.setItem('form', JSON.stringify(form));
 }
-s
\ No newline at end of file
+
+const addButton = document.getElementById('addBook');
+const newTitle = document.getElementById('bookTitle');
+const newAuthor = document.getElementById('authorName');
+const container = document.querySelector('.bodyh1');
+
+addButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (newAuthor.value !== '' && newTitle.value !== '') {
+    const title = newTitle.value;
+    const author = newAuthor.value;
+    const newBook = {
+      title,
+      author,
+    };
+
+    books.push(newBook);
+    storeForm(books);
+  }
+  addButton.form.reset(); // use form property of the button element to reset the form
+});
```

Comment:
- [ ] Great work implementing the `addbook()` functionality while it works fine their a few issues we need to fix in other for it to be better. The issue with your code is that anytime a user adds a book they have to refresh the page in other for them to see the book is added on the UI which isn't a good user experience. 

- While trying to track the issue and know why that is happening I released that the `code` that `displays` the books on the `UI` is executed only once when the page loads `line(40 - 43)`. This `code` block is outside of the event listener that adds new books to the `localStorage`. Therefore, whenever a new book is added, it is added to the `localStorage` but not displayed on the `UI`.

- To fix this issue, the code that displays the books should be run inside the event listener that adds new books to the `localStorage`. This way, whenever a new book is added, it is also displayed on the `UI`. You can do that by adding the `code` that displays the books on the `UI` below the line `35`. Example:
<img width="734" alt="Screenshot 2023-04-28 at 06 35 19" src="https://user-images.githubusercontent.com/107192076/235072848-a67a2735-cac4-436f-9b1b-06befd79f8a5.png">

Reply:
fixed thank you